### ISSUE TITLE
192.168.64.3:8443 may be blocked by firewall rules

### ISSUE THREAD

**Issue by pivotghub** [minishift/minishift#3223]
### General information

  * Minishift version: minishift v1.17.0+f974f0c
  * OS: Linux / macOS / Windows macOS Mojave
  * Hypervisor: KVM / xhyve / Hyper-V / VirtualBox .  xhyve

### Steps to reproduce

  1. minishift start
  2. 
  3. -- Starting profile 'minishift'
-- Checking if https://github.com is reachable ... OK
-- Checking if requested OpenShift version 'v3.9.0' is valid ... OK
-- Checking if requested OpenShift version 'v3.9.0' is supported ... OK
-- Checking if requested hypervisor 'xhyve' is supported on this platform ... OK
-- Checking if xhyve driver is installed ... 
   Driver is available at /usr/local/bin/docker-machine-driver-xhyve
   Checking for setuid bit ... OK
-- Checking the ISO URL ... OK
-- Checking if provided oc flags are supported ... OK
-- Starting local OpenShift cluster using 'xhyve' hypervisor ...
-- Starting Minishift VM .............. OK
-- Checking for IP address ... OK
-- Checking for nameservers ... OK
-- Checking if external host is reachable from the Minishift VM ... 
   Pinging 8.8.8.8 ... FAIL
   VM is unable to ping external host
-- Checking HTTP connectivity from the VM ... 
   Retrieving http://minishift.io/index.html ... FAIL
   VM cannot connect to external URL with HTTP
-- Checking if persistent storage volume is mounted ... OK
-- Checking available disk space ... 8% used OK
-- OpenShift cluster will be configured with ...
   Version: v3.9.0
Deleted existing OpenShift container
Using Docker shared volumes for OpenShift volumes
Using 192.168.64.3 as the server IP
Starting OpenShift using openshift/origin:v3.9.0 ...
-- Starting OpenShift container ... 
   Starting OpenShift using container 'origin'
   Waiting for API server to start listening
FAIL
   Error: timed out waiting for OpenShift container "origin" 
   WARNING: 192.168.64.3:8443 may be blocked by firewall rules
   Details:
     No log available from "origin" container

   Solution:
     Ensure that you can access 192.168.64.3:8443 from your machine
  4. 

### Expected
Minishift should be up and running . Should be accessible through console

### Actual
 Starting OpenShift using container 'origin'
   Waiting for API server to start listening
FAIL
   Error: timed out waiting for OpenShift container "origin" 
   WARNING: 192.168.64.3:8443 may be blocked by firewall rules
   Details:
     No log available from "origin" container

   Solution:
     Ensure that you can access 192.168.64.3:8443 from your machine

### Logs

You can start Minishift with `minishift start --show-libmachine-logs -v5` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.

**Comment by pivotghub** [comment]
[minishifterror.log](https://github.com/minishift/minishift/files/3019610/minishifterror.log)

**Comment by gbraad** [comment]
Can you detail more about what is actually reported? If firewall rules are
a problem, please modify them as we are not able to handle each and
everyone's custom setup.

**Comment by gbraad** [comment]
Duplicate of #3217
